src/docs/Foundations/Typography/BodyStyles/index.mdx

```diff
@@ -1,27 +1,46 @@
 ---
 title: 'Body styles'
 order: 4
+description: 'A body style in Mozaic is the paragraph of a pattern or a page. It must be easy to identify and provide basic typographic styles for multilines texts'
 status:
   sketch: 'wip'
   scss: 'wip'
 ---
 
-> The body patterns provide basic typographic styles for readable multilines text or paragraphs like a product description or an article.
+> A body style in Mozaic is the paragraph of a pattern or a page. It must be easy to identify and provide basic typographic styles for multilines texts.
```

Comment:
A body style provide basic typographic styles (font-size and line-height) for paragraphs (or multilines texts).